R/data.R

```diff
@@ -67,8 +77,13 @@ write_stan_json <- function(data, file) {
           is.data.frame(var) || is.list(var))) {
       stop("Variable '", var_name, "' is of invalid type.", call. = FALSE)
     }
+    if (anyNA(var)) {
+      stop("Variable '", var_name, "' has NA values.", call. = FALSE)
+    }
 
-    if (is.logical(var)) {
+    if (is.table(var)) {
+      var <- unclass(var)
```

Comment:
this removes the "table" class and the resulting object depending on the dimensions of the table (e.g., a 2-D table will become a matrix).